addons/website_livechat/tests/test_livechat_request.py

```diff
@@ -74,19 +79,23 @@ def _common_chat_request_flow(self):
         self.assertEqual(len(channel.message_ids), 3)
         self.assertEqual(channel.message_ids[0].author_id, self.env.ref('base.partner_root'), "Odoobot must be the sender of the 'left the conversation' message.")
         self.assertIn(f"Visitor #{channel.livechat_visitor_id.id}", channel.message_ids[0].body)
-        self.assertEqual(channel.livechat_active, False, "The livechat session must be inactive as the visitor sent his feedback.")
+        self.assertTrue(channel.livechat_end_dt, "The livechat session must be inactive as the visitor sent his feedback.")
 
         return channel
 
     def _clean_livechat_sessions(self):
         # clean every possible mail channel linked to the visitor
-        active_channels = self.env['discuss.channel'].search([('livechat_visitor_id', '=', self.visitor.id), ('livechat_active', '=', True)])
+        active_channels = self.env["discuss.channel"].search(
+            [("livechat_visitor_id", "=", self.visitor.id), ("livechat_end_dt", "=", False)]
+        )
```

Comment:
This search is repeated in so many places, I wonder if we shouldn't have a getter somewhere (either directly in the model or in test files).

Reply:
Lots to do before the freeze, I'd rather improving tests in another PR.